server/keyshare/tasks/tasks_test.go

```diff
@@ -125,8 +154,19 @@ func TestExpireAccounts(t *testing.T) {
 	require.NoError(t, err)
 
 	th.expireAccounts()
+	deleteOnMap := getDeleteDates(t, db)
+
+	assert.Equal(t, 10, countRows(t, db, "users", "delete_on IS NOT NULL"))
+	assert.Equal(t, 4, countRows(t, db, "users", "delete_on IS NULL"))
 
-	assert.Equal(t, 1, countRows(t, db, "users", "delete_on IS NOT NULL"))
+	time.Sleep(1 * time.Second)
+	th.expireAccounts()
+
+	for i, v := range deleteOnMap {
```

Comment:
So `i` is here a unix timestamp right? If I read it I would expect it is an index. That confused me a bit.

Reply:
Changed to `date, count`